packages/ui5-middleware-onelogin/sample/example_simpleproxy_and_onelogin/ui5.yaml

```diff
@@ -0,0 +1,34 @@
+specVersion: "2.2"
+metadata:
+  name: ui5-app
+type: application
+resources:
+  configuration:
+    paths:
+      webapp: ../../../ui5-app/webapp
+
+framework:
+  name: OpenUI5
+  version: "1.95.0"
```

Comment:
Version mismatch

Reply:
changed to 1.97, however the ui5-app is using 1.95 as well. So maybe that should be changed as well?